conjugate_map/conjCalcFunctions.py

```diff
@@ -1,251 +1,306 @@
                 except Exception as e:
                     print(e)
                     continue
             # print(type(lon))
-            if lon>180: lon = lon-360
+            if lon > 180:
+                lon = lon-360
             try:
-                [clat, clon] = findconj(lat, lon, dtime, is_verbose = is_verbose, method = method)
-                if is_verbose: print('Conjugate latitude and longitude: '), print([clat, clon])
-                gdf.loc[index, 'PLAT'],gdf.loc[index, 'PLON'] = [clat, clon]
+                [clat, clon] = findconj(lat, lon, dtime, is_verbose=is_verbose,
+                                        method=method)
+                if is_verbose:
+                    print('Conjugate latitude and longitude: ')
+                    print([clat, clon])
+                gdf.loc[index, 'PLAT'], gdf.loc[index, 'PLON'] = [clat, clon]
             except Exception as e:
                 print('Ran into a problem with ' + index)
                 print(e)
 
-
             # Figure out what coordinates we ultimately want to plot:
-            if lat>0:
-                if is_verbose: print('Setting Northern hemisphere for GLAT of ' + str(lat) + ' on station ' + index)
+            if lat > 0:
+                if is_verbose:
+                    print('Setting Northern hemisphere for GLAT of ' + str(lat)
+                          + ' on station ' + index)
                 gdf.loc[index, 'Hemisphere'] = 'N'
-                if((mode == 'N2S') or (mode == 'flip')): 
+                if (mode == 'N2S' or mode == 'flip'):
                     gdf.loc[index, 'PLAT'] = clat
                     gdf.loc[index, 'PLON'] = clon
                 else:
                     gdf.loc[index, 'PLAT'] = lat
                     gdf.loc[index, 'PLON'] = lon
 
             else:
-                if is_verbose: print('Setting Southern hemisphere for GLAT of ' + str(lat) + ' on station ' + index)
+                if is_verbose:
+                    print('Setting Southern hemisphere for GLAT of ' + str(lat)
+                          + ' on station ' + index)
                 gdf.loc[index, 'Hemisphere'] = 'S'
-                if((mode == 'S2N') or (mode == 'flip')): 
+                if (mode == 'S2N' or mode == 'flip'):
                     gdf.loc[index, 'PLAT'] = clat
                     gdf.loc[index, 'PLON'] = clon
                 else:
                     gdf.loc[index, 'PLAT'] = lat
                     gdf.loc[index, 'PLON'] = lon
-            
+
         except Exception as e:
             print('Ran into a problem with ' + str(index))
             print(e)
             continue
-        
+
         if is_saved:
-            filename = name + '_' + mode + '-'+ method + '-'+ str(dtime)
-            gdf.to_csv(directory + filename +'.csv') #save as .csv
+            filename = name + '_' + mode + '-' + method + '-' + str(dtime)
+            gdf.to_csv(directory + filename + '.csv')  # save as .csv
```

Comment:
```suggestion
            gdf.to_csv(os.path.join(directory, ''.join([filename, '.csv'])))  # save as .csv
```

This will handle any style of directory ending by adding a directory separator if needed for the correct OS.